### ISSUE TITLE
doesn't work without an internet connection :(

### ISSUE THREAD

**Issue by LsdGit** [karlis-vagalis/circular-timer-card#15]
doesn't work without an internet connection

**Comment by helmo** [comment]
This is probably because of the JS modules being imported via CDN as mentioned in the description.
The first 3 lines of the js file do this import.

I agree that it would be nice to have this run independently.

**Comment by alakdae** [comment]
yes, please move the JS so they are locally, it's really annoying

**Comment by alakdae** [comment]
Did it myself, now just need @karlis-vagalis to merge it and release
